queue_job/data/queue_data.xml

```diff
@@ -10,6 +10,15 @@
             <field name="state">code</field>
             <field name="code">model.requeue_stuck_jobs()</field>
         </record>
+        <record id="ir_cron_queue_job_fail_dead_jobs" model="ir.cron">
+            <field name="name">Take care of unresponsive jobs</field>
```

Comment:
While thinking of a better name for this cron I wonder.... why don't we add another method and use only one cron?

Eg:

```
<field name="code">model.gc_stuck_jobs()</field>
[...]

def gc_stuck_jobs(self, started_delta=None, force_low_delta=False):
    self.requeue_stuck_jobs()
    if started_delta:
        self.gc_dead_jobs(started_delta, force_low_delta=force_low_delta)
```

WDYT?